handlers/app.go

```diff
@@ -0,0 +1,48 @@
+package handlers
+
+
+import (
+
+	"github.com/knative-scout/app-api/models"
+	"go.mongodb.org/mongo-driver/bson"
+	"net/http"
+	"github.com/gorilla/mux"
+	"fmt"
+)
+
+// AppByIDHandler returns a single app by ID from the database
+type AppByIDHandler struct {
```

Comment:
Can you rename the file? From `handlers/app.go` to something like `handlers/appbyid.go`